lyricsviewer.go

```diff
@@ -193,9 +200,14 @@ func (l *LyricsViewer) updateSpacerSize(size fyne.Size) {
 		return // renderer not created yet
 	}
 
-	ht := size.Height / 2
-	if l.ActiveLyricPosition == ActiveLyricPositionUpperMiddle {
+	var ht float32
+	switch l.ActiveLyricPosition {
+	case ActiveLyricPositionUpperMiddle:
 		ht = size.Height / 3
+	case ActiveLyricPositionMiddle:
+		ht = size.Height / 2
+	case ActiveLyricPositionTop:
+		ht = l.newLyricLine("W", true).MinSize().Height
```

Comment:
the math for scroll distance is different in the Top case. In the other 2, it's aligning the middle of a lyric line (which may wrap) at a specific position on the widget. With Top, it'd want to align the *top* of the lyric line with the top of the widget. So the scroll math would be based on the height of the current line, as opposed to half the height of the current line + half the height of the next, as is done currently